qpc_tools/utils.py

```diff
@@ -86,7 +91,12 @@ def log_args(args):
 
 
 def create_ansible_command(namespace_args, playbook):
-    """Build Ansible Command."""
+    """Build Ansible Command.
+
+    :param namespace_args: arguments passed in by the user
+    :param playbook: name of the playbook
+    :returns:  The ansible command that will be run
```

Comment:
Ooops Thanks for filling out the doc string here.

Reply:
😄